examples/src/fibonacci/fib2/air.rs

```diff
@@ -18,8 +18,10 @@ pub struct FibAir {
 }
 
 impl Air for FibAir {
+    type AuxRandElements<E: Send + Sync> = ();
     type BaseField = BaseElement;
     type PublicInputs = BaseElement;
+    type AuxProof = ();
```

Comment:
minor nit: I'd probably order these as follows:
```Rust
type BaseField = BaseElement;
type PublicInputs = BaseElement;
type AuxRandElements<E: Send + Sync> = ();
type AuxProof = ();
```
(relevant for other example files as well)

Reply:
Fixed them all